.github/CODEOWNERS

```diff
@@ -1,8 +1,8 @@
 # CODEOWNERS
 # How this file is used? See: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-/docker/*.json @glicht @anara123
-/docker/verify_licenses.py @glicht @anara123
```

Comment:
Removed `@glicht` because the file was invalid as the user has no access to the repo.